docs/docs/writing-hello-world.md

```diff
@@ -1,45 +1,47 @@
 # Writing the sample enclave
 
 ## Prerequisites
-This tutorial assumes that you have completed the [Running your first enclave](running-hello-world.md) tutorial.
+* Complete the [Running your first enclave](running-hello-world.md) tutorial.
 
 ## Introduction
 
-In this tutorial, we will cover the following:
+Use this tutorial to:
 
-* Setting up a new Conclave project using [Conclave Init](conclave-init.md).
-* Modifying the files to create an implementation of the Conclave *sample app* which we explored in the
+* Set up a new Conclave project using [Conclave Init](conclave-init.md).
+* Modify the files to create an implementation of the Conclave *sample app* which we explored in the
   [previous tutorial](running-hello-world.md).
 
-We will be leveraging Conclave's built-in web host to write our Conclave app. To learn more about this, see
-[Conclave web host](conclave-web-host.md). You may want to implement your own host if you need custom functionality or
-you want to incorporate it into an existing server application of some kind. To learn more about that, see
-[Writing your own enclave host](writing-your-own-enclave-host.md).
+This tutorial uses [Conclave web host](conclave-web-host.md) to write a Conclave app.
 
-The client will be a command line app. This is just for simplicity, and you are free to implement
-your client as a GUI app or integrate it with some other program.
+The client is a command line app for simplicity. You can implement your client as a GUI app or integrate it with 
+another program.
 
 ## Project Setup
-Start by creating a new Conclave project using [Conclave Init](conclave-init.md).
+
+To set up a project:
+
+1. Download the Conclave Init tool.
 ```bash
-java -jar <path to sdk>/tools/conclave-init.jar \
+   wget https://repo1.maven.org/maven2/com/r3/conclave/conclave-init/1.4/conclave-init-1.4.jar -O conclave-init.jar
```

Comment:
First of all, the URL needs to be pointing to 1.3, not 1.4, as that is what the docsite will go live on.

Secondly, I would rather that we didn't duplicate the download URL from conclave-init.md. The reason for this is that this URL needs to be manually updated as part of the release process. Would it be possible to say,

`follow the instructions to download Conclave [here](conclave-init.md)` and link to that page? 

However I understand that it is better user experience to not have to tell the user to get somewhere else to download the tool, and then come back and continue with the instructions. So perhaps it's fine to have the URL here as well. We can also update the release process instructions to update the URL on two pages!

Reply:
I have changed the link to 1.3.

I feel it's better to have the URL here.

Reply:
Can you update https://r3-cev.atlassian.net/wiki/spaces/CBU/pages/4123394057/GA+Release+Process to mention this  page as well.